src/stories/autosuggest-text/autosuggest-text.scss

```diff
@@ -0,0 +1,28 @@
+.suggest-text {
```

Comment:
I think, as I have understood BEM, container ought to be an element and not a modifier? and then nest the item inside the container...

Reply:
Perhaps also rename to "autosuggest-text" so it fits the react component? I think this is difficult though, to have names that fit components, as there will be another auto-suggest-component in the dpl-react repo which would also have to have the same name if we would like to enforce consistency in naming?

Reply:
@sinejespersen perf, I adjusted it.

And yes, the same class naming will also be taken over to dpl-react, but by design we hopefully won't be creating any new css in that repo.